spread/project.go

```diff
@@ -64,6 +64,10 @@ type Backend struct {
 	Location string
 	Storage  Size
 
+	// Only for Openstack so far
+	Network        string
```

Comment:
This can be any string like: `network_production-router1` or `network_misc-some-juju-env`

Reply:
The feedback from Gustavo here is that this needs to be a slice []string because people may attach machines to multiple networks (in the general case)